associates/nicole-sump.json

```diff
@@ -0,0 +1,4 @@
+{
+"first": "Nicole"
```

Comment:
Oops, there's a missing comma here. Can you please add that, commit it, then push to your branch again?